udata_recommendations/templates/dataset-recommendations.html

```diff
@@ -9,9 +9,9 @@ <h3>Réutilisations recommandées</h3>
             </span>
         </div>
     </div>
-    <div id="dataset-recommendations-container" class="card-list card-list--columned">
+    <div id="dataset-recommendations-container" class="py-md">
         {% for reuse in reco_reuses %}
-            <div class="col-xs-12 col-md-6"
+            <div class="card reuse-card border p-sm unstyled w-100 mb-md"
```

Comment:
Not convinced about this style, as it doesn't seem consistent with the vertical reuse card layout. Maybe use the same classes as [here](https://github.com/etalab/udata-gouvfr/blob/master/udata_gouvfr/theme/gouvfr/templates/dataset/display.html#L365).

Reply:
I changed to style to have the best of both horizontal and vertical worlds :p